components/ArticleMenu/ArticleMenu.tsx

```diff
@@ -139,7 +139,7 @@ const ArticleMenu = ({
               }}
             >
               <HeartIcon
-                className={`w-6 h-6${
```

Comment:
How come you needed to change the order of these?

Reply:
This is part of an already merged change, I had done a git pull from upstream after committing my changes and since I had global rebase to true, it caused this commit to be moved ahead of my commit.

[Reference to the merged commit](https://github.com/codu-code/codu/commit/871253f2f73106429f40e28fac42f24ba3a04c97)

Reply:
Ok this makes sense. Thanks for the clarification